events/syscall/disallowed_ssh_connection_non_standard_port.go

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+	"github.com/falcosecurity/event-generator/events"
+	"os/exec"
+)
+
+var _ = events.Register(DisallowedSSHConnectionNonStandardPort)
+
+func DisallowedSSHConnectionNonStandardPort(h events.Helper) error {
+	path, err := exec.LookPath("ssh")
+	if err != nil {
+		// If we don't have an SSH, just bail
+		return &events.ErrSkipped{
+			Reason: "ssh utility not found in path",
+		}
+	}
+	// non_standard_port : 443
+	cmd := exec.Command("timeout", "1s", path, "user@example.com", "-p", "443")
```

Comment:
@h4l0gen 

Generally speaking, it is good practice to use [example domains](https://www.iana.org/help/example-domains). 


Just a doubt: will it work with `localhost` as well? :thinking: 

If so, it would be preferable, to avoid outbound traffic. Otherwise, it's ok as-is now.

Reply:
@leogr let me check..

Reply:
@leogr Its not working on localhost :)
![image](https://github.com/falcosecurity/event-generator/assets/117383853/3c537532-f0a4-41e8-be41-143b202d3572)

Reply:
@FedeDP what do you think about using `example.net` ?

Reply:
I think `localhost` should work just fine; in your case it is not working because you are actually succeeding in logging in (ie: you have an ssh daemon on port 443(?))
I'd try to connect to localhost to an higher port, like 60000, and see if that triggers the rule.

Reply:
Uh nope, the rule triggers only for `outbound` connections, therefore it won't fire for localhost. See: https://github.com/falcosecurity/rules/blob/main/rules/falco_rules.yaml#L1223C5-L1223C13

I guess it's ok then.

Reply:
Yes, @FedeDP, if everything seems good, then we are good to proceed with this PR?